### DISCUSSION TITLE
Penpot PWA?

### DISCUSSION THREAD

**Discussion by AmrUIUXD** [uxbox/uxbox#942] [Q&A]
With PWA, Penpot can work offline, access local fonts, perform better because it utilizes device hardware capabilities, right?
Thanks In Advance!

**Comment by hirunatan** [comment]
Yes, indeed, @AmrUIUXD! The possibility of configuring penpot as a PWA is a real option. We are already discussing issues like offline work, local files access and the like, although they are not very mature themes right now.

Check our backlog for more news https://tree.taiga.io/project/penpot/backlog
